symbols/overlay29.yml

```diff
@@ -1978,7 +2239,28 @@ overlay29:
         Gets the lower 2 bytes of a 4-byte number and interprets it as a signed short.
         
         r0: 4-byte number x
-        return: (short)x
```

Comment:
Doesn't really matter much, but I was envisioning `(short)x` as a cast operation. I don't think this needs changing.

Reply:
I thought it was usual to put a space between the casting operator and the variable, is that not the case?

Reply:
I guess it's probably a style convention thing, but I usually see it without a space.